### DISCUSSION TITLE
[WP4147 Discussion]: why PTF1A positively regulate NKX6.1 and PROX1?

### DISCUSSION THREAD

**Discussion by Witiy** [wikipathways/wikipathways-help#139] [pathway-discussion]
### Discussion

Hi, I am a rookie in studying pancreatic cells development. I wander is there any experimental evidence shows that PTF1A positively regulate NKX6.1 and PROX1? It seems that NKX6.1 and PTF1A are antagonistic regulators. However, in WP4147, it does records PTF1A -> NKX6.1. 

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Hi @Witiy, opening the pathway model in [PathVisio](https://pathvisio.org/) (the drawing tool for WikiPathways) will show individual references for interactions and entities. The PTF1A - NKX6.1 and PTF1A - PROX1 interactions have the same reference: https://pmc.ncbi.nlm.nih.gov/articles/PMC2418581/ (see paragraph 5 in Results and Discussion).
